fastlane/lib/fastlane/actions/spaceship_logs.rb

```diff
@@ -0,0 +1,132 @@
+module Fastlane
+  module Actions
+    class SpaceshipLogsAction < Action
+      def self.run(params)
+        latest = params[:latest]
+        print_contents = params[:print_contents]
+        print_paths = params[:print_paths]
+        copy_to_path = params[:copy_to_path]
+        copy_to_clipboard = params[:copy_to_clipboard]
+
+        # Get log files
+        files = Dir.glob("/tmp/spaceship*.log").sort_by { |f| File.mtime(f) }.reverse
+
+        # Filter to latest
+        if latest
+          files = [files.first]
+        end
+
+        # Print contents
+        if print_contents
+          files.each do |file|
+            data = File.read(file)
+            puts("-----------------------------------------------------------------------------------")
+            puts(" Spaceship Log Content - #{file}")
+            puts("-----------------------------------------------------------------------------------")
+            puts(data)
+            puts("\n")
+          end
+        end
+
+        # Print paths
+        if print_paths
+          puts("-----------------------------------------------------------------------------------")
+          puts(" Spaceship Log Paths")
+          puts("-----------------------------------------------------------------------------------")
+          files.each do |file|
+            puts(file)
+          end
+          puts("\n")
+        end
+
+        # Copy to a directory
+        if copy_to_path
+          require 'fileutils'
+          FileUtils.mkdir_p(copy_to_path)
+          files.each do |file|
+            FileUtils.cp(file, copy_to_path)
+          end
+        end
+
+        # Copy contents to clipboard
+        if copy_to_clipboard
+          string = files.map { |file| File.read(file) }.join("\n")
+          require 'open3'
+          Open3.popen3('pbcopy') { |input, _, _| input << string }
```

Comment:
Maybe we could even use [`clipboard`](https://docs.fastlane.tools/actions/clipboard) action? 🤔